src/util/events.js

```diff
@@ -49,12 +56,6 @@ export function getFilteredEvents(events, filters, zipcodes) {
     }
 
     if (filters.zipcode) {
-      // We do not yet have valid zipcodes to filter against,
-      // give it a pass.
-      if (!zipcodes || !zipCodesLength) {
-        return true;
-      }
-
       const zipcodeLatLng = L.latLng(zipcodes[filters.zipcode]);
```

Comment:
...yeah it does, corresponding changes in this PR show a UI message for that case too now:

![image](https://cloud.githubusercontent.com/assets/1724544/24589962/7582ad12-1798-11e7-81ec-3e13045f5459.png)